[2022-02-03 20:53:39,138] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:39,399] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:39,408] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:39,416] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 20:53:39,420] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:39,978] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-10-01 00:00:00+00:00
[2022-02-03 20:53:40,025] {standard_task_runner.py:52} INFO - Started process 6890 to run task
[2022-02-03 20:53:40,107] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_5', 'download_dataset_task', 'scheduled__2019-10-01T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmpgbcy7bcl', '--error-file', '/tmp/tmp6hzqrn2w']
[2022-02-03 20:53:40,127] {standard_task_runner.py:77} INFO - Job 46: Subtask download_dataset_task
[2022-02-03 20:53:41,249] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:53:42,242] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_5
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-10-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-10-01T00:00:00+00:00
[2022-02-03 20:53:42,272] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:53:42,275] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:53:42,334] {subprocess.py:85} INFO - Output:
[2022-02-03 20:53:42,432] {subprocess.py:89} INFO - /tmp/***tmp3p80fa60
[2022-02-03 20:53:42,442] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:53:44,559] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_5, task_id=download_dataset_task, execution_date=20191001T000000, start_date=20220203T205339, end_date=20220203T205344
[2022-02-03 20:53:44,851] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:53:45,672] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
